pkgs/build-support/kernel/make-initrd-ng.nix

```diff
@@ -72,8 +74,11 @@ in
   passAsFile = ["contents"];
   contents = lib.concatMapStringsSep "\n" ({ object, symlink, ... }: "${object}\n${if symlink == null then "" else symlink}") contents + "\n";
 
-  nativeBuildInputs = [makeInitrdNGTool patchelf cpio] ++ lib.optional makeUInitrd ubootTools;
-} ''
+  nativeBuildInputs = [makeInitrdNGTool patchelf cpio] ++ lib.optional makeUInitrd ubootTools ++ lib.optional strip binutils;
+
+} // lib.optionalAttrs strip {
+  STRIP = "${(binutils.nativeDrv or binutils).targetPrefix}strip";
```

Comment:
This should be using lib.optionalString instead of attr merging.

Reply:
No it shouldn't. It needs to be *unset* for the program to work, not empty.

Reply:
I think setting it to `null` would make it unset in the stdenv, but don't quote me on that.

Reply:
@K900 That appears to be correct.

Reply:
Applied.